### PR TITLE
Updated Trainer, Training and TrainerController

### PR DESCRIPTION
**Pull request by eliritt**

### CODE REVIEW

File: src/main/java/model/events/Training.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
 package model.events;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
 
 import model.persons.Trainer;
 
 public class Training {
+	public static final List <Training> trainings = new ArrayList <>();
```
Comment (ghost) [inline src/main/java/model/events/Training.java:10]: Please take a look at the UML diagram. I think it is not good to implement the list with all the trainings in the Training-class itself. Better take this to TrainerController?

File: src/main/java/model/events/Training.java [hunk 2]
```diff
@@ -14,17 +17,22 @@
...
 		this.name = name;
 		this.description = description;
+		this.trainer = trainer;
 		this.dateTime = dateTime;
 		this.place = place;
 		this.maxParticipants = maxParticipants;
 		this.requirements = requirements;
 	}
 	
+	public static List<Training> getTrainings() {
```
Comment (ghost) [inline src/main/java/model/events/Training.java:32]: See above - I would implement this in controller.

**Review by ghost — CHANGES_REQUESTED** [review]
Please take a look at the updated UML diagram and implement it like this. The class Training.java should contain a Set with all the participants while the list of trainings should be implemented in Controller itself...?

**Comment by BeWie24** [comment]
good job! thanks!

**Review by BeWie24 — APPROVED** [review]

**Pull request merged**